game-core/src/test/java/tools/map/making/MapPropertyWrapperTest.java

```diff
@@ -55,13 +54,21 @@ public void shouldThrowExceptionWhenFieldDoesNotExist() {
     }
   }
 
-  static class ExampleEmUnderscoreAttachment extends TestAttachment {
+  static class ExampleEmUnderscoreParentAttachment extends TestAttachment {
     private static final long serialVersionUID = 4140595034027616571L;
 
     @SuppressWarnings("checkstyle:MemberName") // "m_" prefix required for test
     int m_intValue;
 
-    ExampleEmUnderscoreAttachment(final String name, final Attachable attachable, final GameData gameData) {
+    ExampleEmUnderscoreParentAttachment(final String name, final Attachable attachable, final GameData gameData) {
+      super(name, attachable, gameData);
+    }
+  }
+
+  static class ExampleEmUnderscoreChildAttachment extends ExampleEmUnderscoreParentAttachment {
+    private static final long serialVersionUID = -2515485990895802645L;
+
+    ExampleEmUnderscoreChildAttachment(final String name, final Attachable attachable, final GameData gameData) {
```

Comment:
I'm desperately waiting for the day those test attachments can be removed because there no longer are m_ members

Reply:
Yeah, I've got a TODO to remove the `m_` support from `MapPropertyWrapper` once the Checkstyle exercise is complete.